data/processed/ashe_place/gbp_gross_median_salary_s_t.csv

```diff
@@ -0,0 +1,273 @@
+year,nuts_id,nuts_year_spec,gbp_gross_median_salary_s_t
+2011,UKC1,2018,32961.0
```

Comment:
I think we need to run autonuts on these (2018 > 2011).

Also, not sure about the indicator name here, the fact that it ends in `_s_t` makes is pretty obscure to me.